README.md

```diff
@@ -53,6 +53,17 @@ steps:
       # Attach the new line separated listed glob patterns to the release.
       release-assets: |
         ./generated/my-asset.tar.gz
+      # Configure the semantic release branches parameter: https://semantic-release.gitbook.io/semantic-release/usage/workflow-configuration#branches-properties
+      # If not specified semantic-release will use its default branches
+      # configuration, as specified in their Javascript API documentation:
+      # https://github.com/semantic-release/semantic-release/blob/master/docs/usage/configuration.md#branches
+      release-branches: '["+([0-9])?(.{+([0-9]),x}).x","master","next","next-major",{"name":"beta","prerelease":"beta"},{"name":"alpha","prerelease":"alpha"}]'
```

Comment:
Is this nessesary?

Reply:
It is the default value...

Reply:
I was curious why this was part of this PR.

Reply:
Oh, good catch! This was a slip, resolved!